modules/price/commerce_price.views.inc

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * @file
+ * Provide Views data for the Commerce Price module.
+ *
+ * @ingroup views_module_handlers
+ */
+
+use Drupal\field\FieldStorageConfigInterface;
+
+/**
+ * Implements hook_field_views_data().
+ *
+ * Views integration for price fields.
+ */
+function commerce_price_field_views_data(FieldStorageConfigInterface $field_storage) {
+  $data = views_field_default_views_data($field_storage);
+  $field_name = $field_storage->getName();
+  foreach ($data as $table_name => $table_data) {
+    // Amount column.
```

Comment:
We don't need the "Amount column" and "Currency Code column" comments, that's self-evident.

Other than that, this is good to go.